cypress/integration/example.spec.js

```diff
@@ -30,7 +30,7 @@ context('Example', () => {
 
       cy.window().then((win) => {
         win.localStorage.setItem(
-            `@@auth0spajs@@::${client_id}::${audience}::${scope}`,
+            `@@auth0spajs@@::${client_id}::${audience}::${scope}`, // Could also be ::default:: instead of ::${audience}::
```

Comment:
Yep, thanks - if you didn't pass an audience to https://github.com/adamjmcgrath/cypress-spa-example/blob/master/index.html#L14 - this would be `default`;